firmware/lib/STM/STM32F4/STM32F4xx/Include/stm32f4xx.h

```diff
@@ -8,8 +8,8 @@
   *          is using in the C source code, usually in main.c. This file contains:
   *           - Configuration section that allows to select:
   *              - The STM32F4xx device used in the target application
-  *              - To use or not the peripheral�s drivers in application code(i.e. 
-  *                code will be based on direct access to peripheral�s registers 
+  *              - To use or not the peripheral�s drivers in application code(i.e. 
+  *                code will be based on direct access to peripheral�s registers 
```

Comment:
What happend here?

Reply:
I think the file was not encoded as UTF-8, fixed now